src/api/kube.ts

```diff
@@ -2012,7 +2147,34 @@ export class KubeHelper {
     }
   }
 
-  async createCheClusterCertificate(certificateYaml: V1alpha2Certificate, version: string): Promise<void> {
+  async createCertificateIssuer(cheClusterIssuerYamlPath: string, version: string, namespace: string): Promise<void> {
+    const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
+
+    const certificateIssuer = this.safeLoadFromYamlFile(cheClusterIssuerYamlPath)
+    try {
+      await customObjectsApi.createNamespacedCustomObject('cert-manager.io', version, namespace, 'issuers', certificateIssuer)
+    } catch (e) {
+      throw this.wrapK8sClientError(e)
+    }
+  }
+
+  async isCertificateIssuerExists(name: string, version: string, namespace: string): Promise<boolean> {
+    const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
+
+    try {
+      // If issuers doesn't exist an exception will be thrown
+      await customObjectsApi.getNamespacedCustomObject('cert-manager.io', version, namespace, 'issuers', name)
+      return true
+    } catch (e) {
+      if (e.response.statusCode === 404) {
+        return false
+      }
+
+      throw this.wrapK8sClientError(e)
+    }
+  }
+
+  async createCheClusterCertificate(certificateYaml: V1Certificate, version: string): Promise<void> {
```

Comment:
certificateYaml -> certificate. I think here you have javascript object not yaml or path to yaml.